app/uk/gov/hmrc/merchandiseinbaggage/views/Head.scala.html

```diff
@@ -1,23 +0,0 @@
-@*
```

Comment:
This is deleted as two are commented out and unused and moved application.css to Layout